### DISCUSSION TITLE
glightbox issue with inverted color_reg

### DISCUSSION THREAD

**Discussion by gurunanda7** [biati-digital/glightbox#407] [General]
Hi 

I am guru and I am working on creating a portfolio website for me . To show my concepts , I applied the glightbox method as shown in a video https://www.youtube.com/watch?v=UDH--wrJ5eA  .  Unfortunately, at first it didn't work but I managed to find a error and corrected it . But it was not perfect, after clicking the image , image should popup like this 

![Screenshot (250)](https://user-images.githubusercontent.com/126564652/221822796-c94b4db3-5e47-4219-b589-5f81f476f32e.png)

(I copied the exact code and pasted it new file it works! ) but not on original file
but the output is like this with inverted colors in the original file : 

![Screenshot (253)](https://user-images.githubusercontent.com/126564652/221824068-25d6197a-084c-4b36-b6bd-b574e91e862f.png)

Please help me in resolving the issue. 

Thanks

**Comment by gingerchew** [comment]
Without the code you're using, it would be hard to see where the issue. Can you provide the code in a CodePen or repo?
